CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing Guidelines
+
+## Create a grammar
+
+We recommended creating the grammar using our [template], either via the
+"Use this template" button on the web interface, or with the `--template` flag
+of the [gh cli]. Wait a moment for the initialization to complete before cloning
+the repository. The default license is MIT but you can change it to whatever you want.
+
+Make sure to read the Tree-sitter [documentation] before you start
+developing the grammar. The template handles most of the project setup
+for you. Just run these three commands in the grammar directory to get started:
+
+```bash
+npm install --ignore-scripts
+export PATH="$PWD/node_modules/.bin:$PATH"
+```
+
+Or, if you're developing on Windows using PowerShell:
+
+```powershell
+npm install --ignore-scripts
+$env:PATH = "$PWD\node_modules\.bin;$env:PATH"
+```
+
+The template files include several `NOTE:` comments that you should read,
+act upon, and remove afterwards. You can view them all by running this command:
+
+```bash
+grep -Irn NOTE: --exclude-dir=node_modules
+```
+
+## Requirements for inclusion
+
+If you'd like your grammar to be included in this organization you should adhere to the following:
+
+1. Include corpus tests to ensure the parser works as expected.
+2. The external scanner (if any) must be written in C and not C++.
+3. The grammar must be written in JavaScript and not TypeScript or anything else.
+4. The queries must use the last-wins precedence, a-la Neovim style.
+    - _(Optional)_ Include highlight queries and their tests. Other queries are not necessary.
+    - The highlight captures should either use [Tree-sitter][STANDARD_CAPTURE_NAMES]'s
+      "standard" names, or [Neovim][treesitter-highlight-groups]'s names.
+5. You should be somewhat active in maintaining it; if there's no need for maintenance
+   that's totally okay, but you shouldn't let issues pile up for months to years on end.
+6. You must follow [SemVer] for your grammar: breaking changes must result in a major bump,
+   new features in a minor bump, and bug fixes in a patch bump.
+7. _(Optional)_ Commit messages should follow the [Conventional Commits] specification.
```

Comment:
Same. (Also, "should" already implies that it's not unconditionally required.)

Reply:
Lawyer-speak: there's a meaningful difference between "can", "should", and "must", and it helps to use these terms deliberately and consistently in any written guidance.

Reply:
Do we actually want to enforce this?

Reply:
Not sure. But it _does_ make semver easier (especially if you want to automate _that_).

Neovim has a commitlint workflow that does enforce this; it's not universally popular but does help. @dundargoc